matkono/HealthSupApp
# Adding functionality and refactoring screens

## What this PR does

- Add the functionality Back in the tutorial feature;
- Refactor login screen;
- Refactor tutorial screens;
- Remove brackets from error messages;
- Fix the error from radio question for more than 2 options;
- Add validation to email in login screen;

## How to check if this PR is correct

- Check the button 'Anterior' in the tutorial screens;
- Check the background and the text form field in the login features;
- Check layout of the tutorial screens;
- Try to login with a wrong email or password to see the error message;
- Try to select each one of the options in the radio question screen;
- Try to put a incorrect format to email address;

## Evidences: 

- New button 'Anterior' in tutorial's feature:
![image](https://user-images.githubusercontent.com/44846533/91449429-7881f800-e851-11ea-9fa0-12795137ce99.png)

- New login screen:
![image](https://user-images.githubusercontent.com/44846533/91449510-98192080-e851-11ea-968a-dd92a010da27.png)

- New tutorial screen:
![image](https://user-images.githubusercontent.com/44846533/91449667-c3037480-e851-11ea-9c05-679d7ff13a8f.png)

- Fixed error message:
![image](https://user-images.githubusercontent.com/44846533/91449831-f8a85d80-e851-11ea-88f9-94d2ad2f6bd3.png)

- Fixed options in the radio question screen:
![image](https://user-images.githubusercontent.com/44846533/91450075-42914380-e852-11ea-9f04-f4a5a5f96314.png)

- New validation to email address:
![image](https://user-images.githubusercontent.com/44846533/91450140-589f0400-e852-11ea-970a-3e67c4c62029.png)
